apps/app/scripts/reset-project.js

```
@@ -31,77 +31,82 @@ export default function Index() {
-})
+});
 
-const moveDirectories = async (userInput) => {
+const moveDirectories = async userInput => {
   try {
     if (userInput === 'y') {
       // Create the app-example directory
-      await fs.promises.mkdir(exampleDirPath, { recursive: true })
-      console.log(`📁 /${exampleDir} directory created.`)
+      await fs.promises.mkdir(exampleDirPath, { recursive: true });
+      console.log(`📁 /${exampleDir} directory created.`);
```

ESLint

[error] 'console' is not defined.

(no-undef)